src/content/docs/tutorial-optimize-telemetry/detect-anomalies.mdx

```diff
@@ -0,0 +1,160 @@
+
+Fortunately, we have a set of dashboards that you can install [directly from New Relic Instant Observability](https://onenr.io/0EPwJJO9Ow7). The individual dashboards installed by this package include:
+* AWS integrations
+* Azure integrations
+* Google Cloud Platform integrations
+* On-host integrations
+* Kubernetes
+
+<Collapser
+    id="Exercise"
+    title="Checking your data"
+>
+
+Answering the following questions will help you develop confidence in your ability to interpret baseline data and make correct inferences. You can answer these questions using the data ingest baseline and data ingest entity breakdown dashboards. Install those dashboards as described and see how many of these questions you can answer.
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Questions    
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                What is the typical daily ingest rate for the entire organization (all accounts) in the past week?  What was it three months prior?
+            </td>
+        </tr>
+        <tr>
+            <td>
+                What are the top three telemetry types (for the organization as a whole) by ingest?  List each telemetry type and its most recent 30 day ingest rate. 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                How many accounts contribute to this organization's ingest?
+            </td>
+        </tr>
+        <tr>
+            <td>
+                How many accounts (if any) currently contribute more than 50TB per month?  
+            </td>
+        </tr>
+        <tr>
+            <td>
+                What are the top three accounts in terms of ingest for the past 30 days?  
+            </td>
+        </tr>
+        <tr>
+            <td>
+                What is the GB ingest for the calendar month of this past January for the highest consuming account?     
+            </td>
+        </tr>
+        <tr>
+            <td>
+                What are the top three accounts in terms of `ApmEventsBytes` ingest for the past 30 days    
+            </td>
+        </tr>
+        <tr>
+            <td>
+                What is the single largest increase in terms of telemetry type ingest for a given account in the last 9 months? What about decreases?
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Go to the account that contributes the most `ApmEventsBytes` and install/open the `data governance entity breakdown dashboard`.  List the top three APM applications by ingest for the past 24 hours and their respective 24-hour ingest rates.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+| Questions                                                                                                                                                                                                                                       |
```

Comment:
Is this supposed to be here?

Reply:
Yes actually, it's part of this weird table formatting.